apmpackage/apm/changelog.yml

```diff
@@ -29,6 +29,9 @@
     - description: Added tags field mapping to internal_metrics data stream
       type: bugfix
       link: https://github.com/elastic/apm-server/pull/8292
+    - description: Update APM integration description
+      type: bugfix
+      link: https://github.com/elastic/apm-server/pull/8592
```

Comment:
One more note - we should also have a new section `version: 8.3.3` here and add the note to this new section, rather than adding it to the already released `8.3.0` section.

Reply:
Updated :+1: